core/src/main/resources/hudson/PluginManager/sidepanel.groovy

```diff
@@ -22,13 +22,16 @@
  * THE SOFTWARE.
  */
 package hudson.PluginManager
+
+import jenkins.model.Jenkins
+
 l=namespace(lib.LayoutTagLib)
 
 l.header()
 l.side_panel {
     l.tasks {
         l.task(icon:"icon-up icon-md", href:rootURL+'/', title:_("Back to Dashboard"))
-        l.task(icon:"icon-gear2 icon-md", href:"${rootURL}/manage", title:_("Manage Jenkins"), permission:app.CONFIGURE_JENKINS, it:app)
+        l.task(icon:"icon-gear2 icon-md", href:"${rootURL}/manage", title:_("Manage Jenkins"), permission:Jenkins.CONFIGURE_JENKINS, it:app)
```

Comment:
why did you change app by Jenkins ?

Reply:
This was breaking, because app here is "Hudson" class, and that doesn't have the CONFIGURE_JENKINS property

Reply:
ok.